app/routes/languages.rb

```diff
@@ -1,38 +1,46 @@
 require_relative '../../x'
 
 module ExercismWeb
-  module Routes
-    class Languages < Core
-      get '/languages' do
-        tracks = X::Track.all
-        active, inactive = tracks.partition(&:active?)
-        inactive.sort! { |a, b| b.problems.count <=> a.problems.count }
-        erb :"languages/index", locals: { active: active, inactive: inactive }
-      end
+	module Routes
```

Comment:
Would you please have one of your coaches help you figure out auto-indentation in your editor?